python/pyspark/ml/tests.py

```diff
@@ -832,6 +860,96 @@ def test_logistic_regression(self):
         except OSError:
             pass
 
+    def logistic_regression_check_thresholds(self):
+        self.assertIsInstance(
+            LogisticRegression(threshold=0.5, thresholds=[0.5, 0.5]),
+            LogisticRegressionModel
+        )
+
+        self.assertRaisesRegexp(
+            ValueError,
+            "Logistic Regression getThreshold found inconsistent.*$",
+            LogisticRegression, threshold=0.42, thresholds=[0.5, 0.5]
+        )
+
+    def test_binomial_logistic_regression_bounds(self):
```

Comment:
@zero323 We usually run PySpark MLlib test with loading a dataset from ```data/mllib/``` or manual generating a dummy/hard-coded dataset rather than rewrite the same test case as Scala. We keep PySpark test as simple as possible. You can refer [this test case](https://github.com/apache/spark/blob/master/python/pyspark/ml/tests.py#L664). Thanks.